addons/mail/static/tests/legacy/discuss/core/web/chat_window_new_message_tests.js

```diff
@@ -147,3 +147,31 @@ QUnit.test("new message autocomplete should automatically select first result",
     await insertText(".o-discuss-ChannelSelector input", "131");
     await contains(".o-discuss-ChannelSelector-suggestion a.o-mail-NavigableList-active");
 });
+
+QUnit.test('open chat from "new message" chat window should unfold existing window', async () => {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({ name: "John" });
+    pyEnv["res.users"].create({ partner_id: partnerId });
+    pyEnv["discuss.channel"].create({
+        channel_member_ids: [
+            [
+                0,
+                0,
+                {
```

Comment:
Nitpicking but we're trying to avoid hard coded commands by using the `Command` class that exposes the same functions than its python counterpart :)